src/dft/src/architect/ScanArchitectHeuristic.cpp

```diff
@@ -61,12 +137,14 @@ void ScanArchitectHeuristic::architect()
         current_chain->add(std::move(scan_cell));
       }
 
-      // TODO: Perform an actual sorting to reduce the wire length
       current_chain->sortScanCells(
-          [](std::vector<std::unique_ptr<ScanCell>>& falling,
-             std::vector<std::unique_ptr<ScanCell>>& rising,
-             std::vector<std::unique_ptr<ScanCell>>& sorted) {
+          [this](std::vector<std::unique_ptr<ScanCell>>& falling,
```

Comment:
I think if you convert OptimizeScanWireLength to a function you will not need to pass this anymore